Makefile

```diff
@@ -52,20 +52,18 @@ deps: ## Ensures fresh go.mod and go.sum.
 
 .PHONY: generate
 generate: ## Generate all dynamic files.
-generate: generate-pow generate-sol
+generate: pkg/pow/kernelSource.go pkg/contracts/%/%
 
-.PHONY: generate-pow
-generate-pow:
-	@go run ./scripts/opencl
+pkg/pow/kernelSource.go: scripts/opencl/sources/*
```

Comment:
this means: run this target only if `pkg/pow/kernelSource.go` is missing or any of the files in `scripts/opencl/sources/` has changed.